### ISSUE TITLE
Optical keyboard focus missing (BITV 2.4.7a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1168]
All form fields are reachable via tab-key, but there is no optical keyboard focus on the form fields "Wählen Sie das Thema" und "Wählen Sie die Tags". 

https://slidewiki.org/addDeck
